### PR TITLE
Support custom GOBIN in integrations/terraform/Makefile

### PR DESCRIPTION
**Pull request by kopiczko**
Coming from https://github.com/gravitational/teleport/pull/49256

Also related https://github.com/gravitational/protoc-gen-terraform/pull/45

### CODE REVIEW

**Review by marcoandredinis — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

File: integrations/terraform/Makefile [hunk 1]
```diff
@@ -48,12 +48,12 @@ $(BUILDDIR)/terraform-provider-teleport_%: terraform-provider-teleport-v$(VERSIO
...
 # This version must match the version installed by .github/workflows/lint.yaml
 PROTOC_GEN_TERRAFORM_VERSION ?= v3.0.0
-PROTOC_GEN_TERRAFORM_EXISTS := $(shell protoc-gen-terraform version 2>&1 >/dev/null | grep 'protoc-gen-terraform $(PROTOC_GEN_TERRAFORM_VERSION)')
+PROTOC_GEN_TERRAFORM_EXISTS := $(shell $(PROTOC_GEN_TERRAFORM) version 2>&1 >/dev/null | grep 'protoc-gen-terraform $(PROTOC_GEN_TERRAFORM_VERSION)')
 
 .PHONY: gen-tfschema
 gen-tfschema:
 ifndef PROTOC_GEN_TERRAFORM_EXISTS
-	@echo "protoc-gen-terraform $(PROTOC_GEN_TERRAFORM_VERSION) is not installed. Please, refer to README.md for installation instructions."
+	@echo "protoc-gen-terraform $(PROTOC_GEN_TERRAFORM_VERSION) is not installed. Please, see https://github.com/gravitational/protoc-gen-terraform for installation instructions."
```
Comment (hugoShaka) [inline integrations/terraform/Makefile:56]: README.md currently contains the correct installation instructions with the right version, why this change?

Comment (kopiczko) [reply integrations/terraform/Makefile:56]: Sorry, I missed this was the README.md in ./integrations/terraform directory. I looked into the top level one and couldn't find anything.
